testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -950,3 +950,8 @@
 When(/^I select the next maintenance window$/) do
   find(:xpath, "//select[@id='maintenance-window-select']/option", match: :first).select_option
 end
+
+When(/^I enter "([^"]*)" as the redfish server address$/) do |host|
```

Comment:
```suggestion
When(/^I enter the server hostname as the redfish server address$/) do
```

Reply:
In my opinion, if we are already sticking into use it only on the server node, in other steps like the stop.
We shouldn't allow here to pass other values. Just create a sentence without parameters, it's fine.

Reply:
done